### PR TITLE
Update docs to reflect VS2017 release [dev15.0.x]

### PR DESCRIPTION
**Pull request by davkean**
Updated docs to reflect VS2017 release and new branch structure

### CODE REVIEW

**Comment by davkean** [comment]
tag @dotnet/project-system

File: README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
-## Welcome to the Roslyn C# and Visual Basic project system
+## Welcome to the new C# and Visual Basic project system
 
-|Debug|Release|
-|:--:|:--:|
-|[![Build Status](http://ci.dot.net/job/dotnet_roslyn-project-system/job/master/job/windows_debug/badge/icon)](http://ci.dot.net/job/dotnet_roslyn-project-system/job/master/job/windows_debug/)|[![Build Status](http://ci.dot.net/job/dotnet_roslyn-project-system/job/master/job/windows_release/badge/icon)](http://ci.dot.net/job/dotnet_roslyn-project-system/job/master/job/windows_release/)|
+|Branch|Debug|Release|
+|--|:--:|:--:|
+|dev15.0.x|[![Build Status](http://ci.dot.net/job/dotnet_roslyn-project-system/job/dev15.0.x/job/windows_debug/badge/icon)](http://ci.dot.net/job/dotnet_roslyn-project-system/job/dev15.0.x/job/windows_debug/)|[![Build Status](http://ci.dot.net/job/dotnet_roslyn-project-system/job/dev15.0.x/job/windows_release/badge/icon)](http://ci.dot.net/job/dotnet_roslyn-project-system/job/dev15.0.x/job/windows_release/)|
```
Comment (333fred) [inline README.md]: Not quite sure what's wrong, but this doesn't render a table correctly.

File: docs/repo/Getting Started.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ### Code
 
 #### Visual Studio
-From within [Visual Studio "15" Preview 2](https://www.visualstudio.com/en-us/downloads/visual-studio-next-downloads-vs), simply open _src\ProjectSystem.sln_.
+From within [Visual Studio 2017](https://www.visualstudio.com/downloads/), simply open _src\ProjectSystem.sln_.
```
Comment (333fred) [inline docs/repo/Getting Started.md]: It's not quite this simple, you need the VB vsix installed. We need an acquisition story for this outside a private share.

Comment (333fred) [inline docs/repo/Getting Started.md]: Otherwise, you'll get a migration popup.

File: README.md (hunk 1 above)
Comment (davkean) [inline README.md]: What do you mean? Renders okay for me:

image

File: docs/repo/Getting Started.md [hunk 3]
```diff
@@ -17,14 +17,14 @@ This builds, deploys and run tests.
...
+By default when you build inside Visual Studio or the command-line, the project system and other binaries gets deployed to the _RoslynDev_ experimental instance of Visual Studio. They are setup so that they _override_ any binaries that come with Visual Studio.
 
 #### Visual Studio
-By default when you build, the project system and other binaries gets deployed to the _RoslynDev_ experimental instance of Visual Studio. They are setup so that they _override_ any binaries that come with Visual Studio.
 
 To start debugging:
 
 1. Open __src\ProjectSystem.sln__
-2. Right-click on the __ProjectSystemSetup__ project, and choose __Set As Startup Project__
+2. Right-click on the __ProjectSystemDogfoodSetup__ project, and choose __Set As Startup Project__
```
Comment (333fred) [inline docs/repo/Getting Started.md]: Why do we even have `ProjectSystemDogfoodSetup`? I've always run `ProjectSystemSetup`.

File: README.md (hunk 1 above)
Comment (333fred) [inline README.md]: image

File: docs/repo/Getting Started.md (hunk 3 above)
Comment (davkean) [inline docs/repo/Getting Started.md]: To registry the VB selector and CPS trace listener, but I do I want to kill it.

File: docs/repo/Getting Started.md (hunk 2 above)
Comment (davkean) [inline docs/repo/Getting Started.md]: Not much I can do right now about that...

Comment (333fred) [inline docs/repo/Getting Started.md]: But we should document that it's expected, at least

**Comment by davkean** [comment]
I'm going to fix the migration pop-up instead, I'll make it so that the project can open regardless of whether the selector is installed.

**Pull request merged**